tdrs-frontend/src/components/SplashPage/SplashPage.jsx

```diff
@@ -15,18 +17,30 @@ function SplashPage() {
   const authLoading = useSelector((state) => state.auth.loading)
   const isInactive = useSelector((state) => state.auth.inactive)
   const alertRef = useRef(null)
+  const dispatch = useDispatch()
 
   const handleClick = (event) => {
-    event.preventDefault()
-    window.location.href = `${process.env.REACT_APP_BACKEND_URL}/login/oidc`
+    if (
+      !window.location.href.match(/https:\/\/.*\.app\.cloud\.gov/) &&
```

Comment:
nit: This logic gets repeated a few times, making it a great candidate for abstraction. I'd set this in a constants.js file under `src/utils/` or something similar and import where needed.